pages/index.js

```diff
@@ -1,7 +1,40 @@
+import { activities } from "@/db/users/activities/activities";
+// import Link from "next/link";
+import styled from "styled-components";
+import { Header } from "@/components/StyledHeader";
+import { StyleBox } from "@/components/StyledBox";
+import { StyleList } from "@/components/StyledList";
+
+const StyleMain = styled.main`
+  margin-top: 7rem;
+  background-color: beige;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-color: green;
+`;
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <StyleMain>
+      <Header>Adventure list</Header>
+      <h2 className="actions">
+        {activities.map((activity) => {
+          return (
+            <>
+              <StyleBox>
+                <div>
+                  <StyleList>
+                    <li key={activity.id}>
```

Comment:
The `key` needs to be moved from here to the parent element that gets returned from the mapping function. I.e. the element highest up the tree that is still in the map.